client/src/components/questions_answers/Answers.jsx

```diff
@@ -41,7 +41,7 @@ const Answers = ( { questionId } ) => {
       }
     };
     const url = process.env.GIT_API_URL;
```

Comment:
Please remove const url as it is not needed anymore

Reply:
done